### ISSUE TITLE
Count parameter is ignored in batch requests

### ISSUE THREAD

**Issue by tyHadd** [thorning/node-mailchimp#55]
I am trying to make a batch request that outputs 20 member ID's instead of the default ```count``` of 10. However, it seems that my request's ```count``` parameter is being ignored. Other parameters, such as ```fields: "members.id"``` are working as expected.

Using ```count``` works on normal requests but I need to get ```count``` to register on batches since I want to be able to retrieve a large number of contacts from my list without timing out.

```js
mailchimp.batch({
    method: 'GET',
    path: '/lists/550de38790/segments/173/members',
    params: {
        count: "20",
        fields: "members.id"
    }
}, function (error, result) {
    if (error) {
        throw Error(`\nError: \n${error}\n`)
    } else {
        console.log('\nSuccess\n')
        console.log(result)
    }
})
```

```result``` will always be a list of 10 members, regardless of what I set ```count``` to. This is also true when I remove ```count``` altogether from the request.

```js
{ members:
   [ { id: 'bd2d16fc614c3a70dfe42bf9f8da83ec' },
     { id: '95f1ff9181aa755f840b55eb27c20cae' },
     { id: 'a685cc09949a5d1f2a36e1c1e9a187b3' },
     { id: '991ac435ddaca416ff60b7f38b73b8c7' },
     { id: 'b9511a943f7f27a2eb09fa5b3e2164d1' },
     { id: '15a13094f98cc07f97c94668bceb130f' },
     { id: '05070ec914abd2abb6f3d57efb2e533c' },
     { id: 'fd9fb78711c1486e8b96a5ce9029798b' },
     { id: '75b3434be1d4f568f5f689e621e3e524' },
     { id: 'cf609fc93522253ad395fa6d9fdeab73' } ],
  returned_item_count: 10 }
```

Is anyone able to shed some light on why this is happening? It's driving me nuts.

**Comment by thorning** [comment]
You can try to rename the `params` property to `query`. If that works, it should be a quick bug fix (I'll be happy to accept a PR)

**Comment by tyHadd** [comment]
@thorning Yeah, I've tried changing the count to a number and I get the same results.

**Comment by thorning** [comment]
It might be a bug in mailchimps side then. does count work, when you use the non-segment endpoint [members](https://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/#read-get_lists_list_id_members)?
Or does it work if you set it to something lower than 10?

**Comment by tyHadd** [comment]
I've tried setting it lower than 10 and received the same result.

```count``` **does** work on the members endpoint, so this confirms that it must be on MC's end. I've also built the same segment endpoint request in Postman to confirm it won't work there either and it didn't.

I'll try to get in touch with Mailchimp about this. I have a feeling it will take them a bit to get back to me if they do at all, but at least I have a workaround now. Thanks for your help!
